src/Sentry/Extensibility/HubAdapter.cs

```diff
@@ -75,7 +74,7 @@ public void WithScope(Action<Scope> scopeCallback)
         /// Forwards the call to <see cref="SentrySdk"/>.
         /// </summary>
         [DebuggerStepThrough]
-        public ITransaction StartTransaction(
+        public ITransactionTracer StartTransaction(
```

Comment:
This should return `ISpan`

Reply:
on Java, `StartTransaction` still returns `ITransaction` which extends `ISpan` to not do breaking changes, `getSpan` returns `ISpan`, and `startChild` aways returns `ISpan`

that's what makes the most sense for me, in the feature `StartTransaction` could also return `ISpan`

right now this introduces a breaking change by changing the return type, we've agreed to not do that, right? unless `ITransactionTracer` implements  `ITransaction`, didn't check yet